README.md

```diff
@@ -1,7 +1,5 @@
 # moodle-tool_cloudmetrics
 
-[![ci](https://github.com/catalyst/moodle-tool_cloudmetrics/workflows/ci/badge.svg)](https://github.com/catalyst/moodle-tool_cloudmetrics/actions?query=workflow%3Aci)
```

Comment:
Why no badge?

Reply:
The wiki says that's the old way. I assumed it was added automatically if ci.yml was there. My misinterpretation?

Reply:
We still want the badge, the wiki mentions removing the old travis badge if it is present. I've tweaked the wiki